python/ptpython/Portfile

```diff
@@ -1,32 +1,17 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           python 1.0
+PortGroup           obsolete 1.0
 
 name                ptpython
 version             3.0.20
-revision            0
+revision            1
 
 supported_archs     noarch
 platforms           {darwin any}
-license             BSD
-maintainers         nomaintainer
-
-description         Python REPL built on top of prompt_toolkit
-long_description    {*}${description}
-
-homepage            https://github.com/prompt-toolkit/ptpython
 
-checksums           rmd160  bdb635972a91b933f9caa4a647bae4505dcb872f \
-                    sha256  eafd4ced27ca5dc370881d4358d1ab5041b32d88d31af8e3c24167fe4af64ed6 \
-                    size    414821
-
-python.default_version 310
-
-depends_build-append \
-                    port:py${python.version}-setuptools
+replaced_by         py-ptpython
+categories          python
+license             BSD
 
-depends_lib-append  port:py${python.version}-appdirs \
-                    port:py${python.version}-jedi \
-                    port:py${python.version}-prompt_toolkit \
-                    port:py${python.version}-pygments
+# remove after September 10, 202
```

Comment:
Minor nit-pick: Add the full year (2024).  ;-)

Reply:
@mascguy here it is!